### PR TITLE
Updates for VS 2017 version

### PR DESCRIPTION
**Pull request by davidchesnut**
Updates to articles to move them from VS 2015 to VS 2017

### CODE REVIEW

**Comment by o365devx** [comment]
OPS Build status updates of commit _[e31e650](https://github.com/OfficeDev/office-js-docs-pr/pull/624/commits/e31e65014eaada758f651f727b9616bdcc8bde6e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/powerpoint/use-document-themes-in-your-powerpoint-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-morevsupdates/docs/powerpoint/use-document-themes-in-your-powerpoint-add-ins.md) | :white_check_mark:Succeeded | |
[docs/project/create-your-first-task-pane-add-in-for-project-by-using-a-text-editor.md](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-morevsupdates/docs/project/create-your-first-task-pane-add-in-for-project-by-using-a-text-editor.md) | :white_check_mark:Succeeded | |
[docs/project/project-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-morevsupdates/docs/project/project-add-ins.md) | :white_check_mark:Succeeded | |
[docs/word/create-better-add-ins-for-word-with-office-open-xml.md](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-morevsupdates/docs/word/create-better-add-ins-for-word-with-office-open-xml.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201812170153506997-624%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Zc5pNHp2g4%2F05js%2B8pQoUi%2BQFqXKAW7Egn%2FGf2DY07s%3D&st=2018-12-17T01%3A50%3A08Z&se=2019-01-17T01%3A55%3A08Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
OPS Build status updates of commit _[3a10369](https://github.com/OfficeDev/office-js-docs-pr/pull/624/commits/3a10369437fe0d8140f2237ad226bd417763391b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/powerpoint/use-document-themes-in-your-powerpoint-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-morevsupdates/docs/powerpoint/use-document-themes-in-your-powerpoint-add-ins.md) | :white_check_mark:Succeeded | |
[docs/project/create-your-first-task-pane-add-in-for-project-by-using-a-text-editor.md](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-morevsupdates/docs/project/create-your-first-task-pane-add-in-for-project-by-using-a-text-editor.md) | :white_check_mark:Succeeded | |
[docs/project/project-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-morevsupdates/docs/project/project-add-ins.md) | :white_check_mark:Succeeded | |
[docs/word/create-better-add-ins-for-word-with-office-open-xml.md](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-morevsupdates/docs/word/create-better-add-ins-for-word-with-office-open-xml.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201812170157015957-624%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=SwZVxQxjwTGfUz8iOIguCMKwG19qAjNFtYq0ax6rbZ8%3D&st=2018-12-17T01%3A53%3A19Z&se=2019-01-17T01%3A58%3A19Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/powerpoint/use-document-themes-in-your-powerpoint-add-ins.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Every Office document theme defines 12 colors. Ten of these colors are available
 
 ![Color palette](../images/office15-app-color-palette.png)
 
-To view or customize the full set of 12 theme colors in PowerPoint, in the  **Variants** group on the **Design** tab, click the **More** drop-down - then point to **Color**, and click  **Customize Colors** to display the **Create New Theme Colors** dialog box.
+To view or customize the full set of 12 theme colors in PowerPoint, in the  **Variants** group on the **Design** tab, click the **More** drop-down - then choose **Colors** > **Customize Colors** to display the **Create New Theme Colors** dialog box.
```
Comment (kbrandl) [inline docs/powerpoint/use-document-themes-in-your-powerpoint-add-ins.md]: Consider changing "click" -> "select"

File: docs/powerpoint/use-document-themes-in-your-powerpoint-add-ins.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ Every Office document theme also defines two fonts -- one for headings and one f
 
 ![The font picker](../images/office15-app-font-picker.png)
 
-To view or customize theme fonts in PowerPoint, in the  **Variants** group on the **Design** tab, click the **More** drop-down - then point to **Fonts**, and click  **Customize Fonts** to display the **Create New Theme Fonts** dialog box.
+To view or customize theme fonts in PowerPoint, in the  **Variants** group on the **Design** tab, click the **More** drop-down - then choose **Fonts** > **Customize Fonts** to display the **Create New Theme Fonts** dialog box.
```
Comment (kbrandl) [inline docs/powerpoint/use-document-themes-in-your-powerpoint-add-ins.md]: Consider changing "click" -> "select"

File: docs/powerpoint/use-document-themes-in-your-powerpoint-add-ins.md (hunk 1 above)
Comment (davidchesnut) [reply docs/powerpoint/use-document-themes-in-your-powerpoint-add-ins.md]: changed

**Comment by o365devx** [comment]
OPS Build status updates of commit _[69c51b2](https://github.com/OfficeDev/office-js-docs-pr/pull/624/commits/69c51b263c931e32b2f88ca24dbc285f97d1c334)_: 

:clock10: **Preparing**: average preparing time is _45 sec(s)_

File: docs/powerpoint/use-document-themes-in-your-powerpoint-add-ins.md (hunk 2 above)
Comment (davidchesnut) [reply docs/powerpoint/use-document-themes-in-your-powerpoint-add-ins.md]: changed

**Comment by davidchesnut** [comment]
@kbrandl I changed to select. Merging PR. Thanks!

**Comment by o365devx** [comment]
OPS Build status updates of commit _[69c51b2](https://github.com/OfficeDev/office-js-docs-pr/pull/624/commits/69c51b263c931e32b2f88ca24dbc285f97d1c334)_: 

:clock10: **Incremental building**: average incremental building time is _28 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[69c51b2](https://github.com/OfficeDev/office-js-docs-pr/pull/624/commits/69c51b263c931e32b2f88ca24dbc285f97d1c334)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/powerpoint/use-document-themes-in-your-powerpoint-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-morevsupdates/docs/powerpoint/use-document-themes-in-your-powerpoint-add-ins.md) | :white_check_mark:Succeeded | |
[docs/project/create-your-first-task-pane-add-in-for-project-by-using-a-text-editor.md](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-morevsupdates/docs/project/create-your-first-task-pane-add-in-for-project-by-using-a-text-editor.md) | :white_check_mark:Succeeded | |
[docs/project/project-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-morevsupdates/docs/project/project-add-ins.md) | :white_check_mark:Succeeded | |
[docs/word/create-better-add-ins-for-word-with-office-open-xml.md](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-morevsupdates/docs/word/create-better-add-ins-for-word-with-office-open-xml.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C20%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201812200142321741-624%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=pyvvAuPKUp2uuCeA5sgaa2lN9QxfFF98Kvf8cpM1fQc%3D&st=2018-12-20T01%3A38%3A36Z&se=2019-01-20T01%3A43%3A36Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**